objax/module.py

```diff
@@ -191,6 +191,8 @@ def pmap(tensor_list: List[ShardedDeviceArray], random_list: List[ShardedDeviceA
 
     def device_reshape(self, x: JaxArray) -> JaxArray:
         """Utility to reshape an input array in order to broadcast to multiple devices."""
+        assert x.shape[0]%self.ndevices == 0, f'Must be able to equally divide batch {x.shape} among ' \
```

Comment:
Nit: spacing, can you put space around operators to be consistent with rest of the code?
```python
assert x.shape[0] % self.ndevices == 0, f'Must be able to equally divide batch {x.shape} among ' \
                                        f'{self.ndevices} devices, but does not go equally.'
```